pages/osx/awk.md

```diff
@@ -0,0 +1,24 @@
+# awk
+
+> Find and Replace text within file(s).
+> More information: <https://ss64.com/osx/awk.html>.
+
+- Define the input field separator = regular expression fs:
+
+`awk [ -F fs ]`
```

Comment:
```suggestion
- Define an input field separator:

`awk --field-separator {{regular_expression}}`
```

Mandatory fix: remove unsupported syntax: unexpected `[` before `-F`.
Mandatory fix: [long option must be used if it's portable](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax).
Mandatory fix: command syntax is wrong: `--field-separator` doesn't accept one literal value `fs`: placeholder required.

Reply:
[Long options do not exist for `awk` on macOS.](https://keith.github.io/xcode-man-pages/awk.1.html)

Reply:
Suggestion updated.